federation/cluster/common.sh

```diff
@@ -24,7 +24,7 @@ source "${KUBE_ROOT}/cluster/kube-util.sh"
 # kubefed configuration
 FEDERATION_NAME="${FEDERATION_NAME:-e2e-federation}"
 FEDERATION_NAMESPACE=${FEDERATION_NAMESPACE:-federation-system}
-FEDERATION_KUBE_CONTEXT="${FEDERATION_KUBE_CONTEXT:-e2e-federation}"
+FEDERATION_KUBE_CONTEXT="${FEDERATION_KUBE_CONTEXT:-${FEDERATION_NAME}}"
```

Comment:
Yeah, that's a good idea.

Right now we have a path where `FEDERATION_KUBE_CONTEXT` and `FEDERATION_NAME` might not be the same. The plan is to remove that path in 1.7. I will make the recommended change in the PR that removes the alternative path.